stunnel/stunnel.ml

```diff
@@ -134,21 +135,25 @@ let config_file verify_cert extended_diagnosis host port =
     ]
   ; if is_fips then ["fips=yes"] else ["fips=no"]
   ; if extended_diagnosis then ["debug=authpriv.7"] else ["debug=authpriv.5"]
+  ; [ "sslVersion = TLSv1.2"
+    ; "ciphers = " ^ Xcp_const.good_ciphersuites
+    ; "curve = secp384r1"
+    ]
   ; if verify_cert then
-      ["verify=2"
-      ; sprintf "checkHost=%s" host
+      [ ""
+      ; "# use SNI to request a specific cert. CAfile contains"
+      ; "# public certs of all hosts in the pool and must contain"
+      ; "# the cert of the server we connect to"
```

Comment:
I think it's good to have these comments, but probably in the ocaml instead (this config is never actually written to disk)

Reply:
I know that the configuration is transient but still think the place is correct. Either way, would change it if you prefer.